src/utils/formatting.ts

```diff
@@ -7,6 +7,10 @@ export function secondsToTime(seconds: number): string {
     { label: 'min', duration: 60 },
   ]
 
+  if (seconds === 0) {
+    return `0 ${units[units.length - 1].label}`
```

Comment:
formatting does not work when 0 is passed. It tries to map it but the current logic ignores the value when remaining seconds is 0. This means when 0 is passed to the method, an empty string is returned. This check is to return "0 min" for a call when 0 is passed

Reply:
Returning a `falsy` value here was intentional, mostly cause we don't want to show certain stats if the duration is 0. An example can be seen on the Rewind page, where we check `userRewind.duration.user` before rendering a card. In your PR, an error is thrown, because the conditional passes (0 mins is not falsy), but there is nothing to actually show.

I understand the purpose behind this being that we always show the logged in users card, even if there is `0 mins` playtime. I would propose we remove this and add a fallback just to the `<MediaItem />`

```
{/* Duration */}
{activeStats.includes('duration') && (
  <li className='icon-stat-wrapper'>
    <ClockIcon />
    {secondsToTime(data.total_duration) || '0 min'}
  </li>
)}
```